README.rst

```diff
@@ -80,16 +120,22 @@ This section outlines what stages a proposal may go through. The stage is identi
    * `How to comment on a proposal <#how-to-comment-on-a-proposal>`__
    * `≡ List of proposals under discussion <https://github.com/ghc-proposals/ghc-proposals/pulls?q=is%3Aopen+is%3Apr+no%3Alabel>`__
  
-   At some point the proposal author brings the proposal before the committee for review, which will prompt the committee secretary to appoint a shepherd for the proposal.
+4. (No label.)  **Responsibility for next action:** author.
+
+   *Note: The Reviewing process does NOT start automatically when the discussion ends.*
+
+   At some point, the proposal author wishes to start a review process of the proposal by the committee. 
+
+   Then the author MUST explicitly ask in comments the committee secretary to bring the proposal before the committee for review (and to appoint a shepherd for it).
```

Comment:
These are helpful clarifications, thanks. I think they belong as part of the previous "community discussion" stage, though? This isn't a separate stage, but explains how to transition to the next stage.